exchanges/okx/okx_websocket.go

```diff
@@ -385,10 +386,25 @@ func (ok *Okx) handleSubscription(operation string, subscriptions []stream.Chann
 		var algoID string
 		var uid string
 
-		if arg.Channel == okxChannelAccount ||
-			arg.Channel == okxChannelOrders {
+		switch arg.Channel {
```

Comment:
🔥

Reply:
What I noticed during this is that switching implies type conversion.
You can switch an `any` and it'll cast and compare.